app/screens/gallery/footer/index.tsx

```diff
@@ -138,27 +140,18 @@ const Footer = forwardRef<FooterRef, FooterProps>((props: FooterProps, ref) => {
         toggle,
     }), [visible]);
 
-    const translateY = opacity.interpolate({
-        inputRange: [0, 1],
-        outputRange: [99, 0],
-    });
-
-    const downloadingY = downloadingOpacitity.interpolate({
-        inputRange: [0, 1],
-        outputRange: [99, 0],
-    });
-
```

Comment:
why the removal of the animation? I don't think this is expected but let me know your reasoning

Reply:
Also not needed